manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: JiroShimizu.Jw_cad
+PackageVersion: 8.25a
```

```suggestion
PackageVersion: 8.25.1.0
```
This must match Control Panel.